akka-discovery-azure-api/src/main/resources/reference.conf

```
@@ -0,0 +1,60 @@
+######################################################
+# Akka Service Discovery Azure Config                  #
+######################################################
+
+akka.discovery {
+  # Set the following in your application.conf if you want to use this discovery mechanism:
+  # method = aws-api-ec2-tag-based
+  azure-rbac-aks-api {
+    class = akka.discovery.kubernetes.AzureRbacAksServiceDiscovery
+
+    authority-host = "https://login.microsoftonline.com/"
+    authority-host = ${?AZURE_AUTHORITY_HOST}
+
+    // can't be empty
+    client-id = ${AZURE_CLIENT_ID}
+
+    federated-token-file = "/var/run/secrets/azure/tokens/azure-identity-token"
+    federated-token-file = ${?AZURE_FEDERATED_TOKEN_FILE}
+
+    // can't be empty
+    tenant-id = ${AZURE_TENANT_ID}
+
+    # AKS uses a pair of first-party Microsoft Entra applications
+    # The AKS Microsoft Entra server application ID that the server side uses is 6dae42f8-4368-4678-94ff-3960e28e3630
+    entra-server-id = "6dae42f8-4368-4678-94ff-3960e28e3630"
+    entra-server-id = ${?AZURE_SERVER_ID}
+
+    # API server, cert and token information. Currently these are present on K8s versions: 1.6, 1.7, 1.8, and perhaps more
+    api-ca-path = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+    api-token-path = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+    api-service-host-env-name = "KUBERNETES_SERVICE_HOST"
+    api-service-port-env-name = "KUBERNETES_SERVICE_PORT"
```

Could these just inject the value instead of indirection of reading env var in code?

```suggestion
    api-service-host = ${KUBERNETES_SERVICE_HOST}
    api-service-port = ${KUBERNETES_SERVICE_PORT}
```